tests/interfaces-network-bind/task.yaml

```diff
@@ -30,6 +30,8 @@ prepare: |
   EOF
 
 restore: |
+  echo "FIXME: no need to remove snaps when the state is reset in an upper level restore"
```

Comment:
Maybe tweak the text a bit: `FIXME: once the state is properly reset in the upper levels this can go away` or something? I.e. emphasise that the upper levels do not restore correctly currently and we need this hack because of this :)